docs/hosting/custom_en-US.json

```diff
@@ -0,0 +1,182 @@
+        },
+        "DownloadFile": {
+            "downloadWarningText": "If you download this file outside of Parsec, we can no longer guarantee its security.",
+            "noPermissions": "Parsec doesn't have the necessary permissions to save a file on your disk. Please allow file editing permissions on this website if you want to be able to download a file."
+        }
+    },
+    "SideMenu": {
+        "frozen": {
+            "email": "contact{'@'}parsec.cloud"
+        }
+    },
+    "DevicesPage": {
+        "greet": {
+            "subtitles": {
+                "waitForGuest": "Parsec must be open on both devices during the onboarding process."
+            }
+        }
+    },
+    "ExportRecoveryDevicePage": {
+        "subtitles": {
+            "warningUniqueness": "Please note that these files are not stored by Parsec. If you lose one or both, you will need to create a new recovery file."
+        },
+        "filenames": {
+            "recoveryFile": "Parsec_Recovery_File_{org}.psrk",
+            "recoveryKey": "Parsec_Recovery_Code_{org}.txt"
+        }
+    },
+    "JoinOrganization": {
+        "linkFormPlaceholder": "parsec3://example.com/Org?...",
+        "instructions": {
+            "finish": {
+                "first": "You can now log in to Parsec."
+            }
+        },
+        "titles": {
+            "waitForHost": "Welcome to Parsec!"
+        }
+    },
+    "ClaimDeviceModal": {
+        "subtitles": {
+            "claimDevice": "You must have Parsec open on both devices during the onboarding process.",
+            "done": "The device has been created and added to this organization. You can now use it to connect to Parsec."
+        }
+    },
+    "validators": {
+        "invalidProtocol": "Link should start with 'parsec3://'.",
+        "claimLink": {
+            "invalidProtocol": "Link should start with 'parsec3://'."
+        },
+        "claimUserLink": {
+            "invalidProtocol": "Link should start with 'parsec3://'."
+        },
+        "claimDeviceLink": {
+            "invalidProtocol": "Link should start with 'parsec3://'."
+        },
+        "bootstrapLink": {
+            "invalidProtocol": "Link should start with 'parsec3://'."
+        }
+    },
+    "loginPage": {
+        "title": "My Parsec account",
+        "noAccount": "No Parsec account?",
+        "createAccount": {
+            "description": "No Parsec account yet?",
+            "welcome": "Welcome to Parsec",
+            "welcomeDescription": "Your Parsec Share account has been created successfully. You now have access to Parsec."
+        }
+    },
+    "clientArea": {
+        "sidebar": {
+            "noOrganizationDescription": "Create your first organization to start using Parsec."
+        },
+        "orders": {
+            "request": {
+                "userNeeds": {
+                    "label": "How many people would use Parsec?"
+                }
+            }
+        }
+    }
+}
```

Comment:
- Parsec Account not yet released
- Customer Area to be disabled for on-premise

```suggestion
```